app/views/projects/_table.html.erb

```diff
@@ -14,7 +14,9 @@
   </thead>
 
   <tbody>
+
     <% projects.each do |project| %>
+    <% cache [project.id, @result] do %>
```

Comment:
Feel free to move the individual row to a separate fragment, for separate rendering.